pom.xml

```diff
@@ -80,6 +80,21 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.build.resourceEncoding>UTF-8</project.build.resourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <!-- docker build support -->
+        <GIT_COMMIT>unknown</GIT_COMMIT>
+        <BUILD_NUMBER>-1</BUILD_NUMBER>
+        <docker.skip-build>true</docker.skip-build>
+        <docker.skip-test>true</docker.skip-test>
+        <docker.registry>placeholder/</docker.registry>  <!-- Registry for this build. Must specify a value, or exec plugin throws NPE :((( -->
+        <docker.tag>${project.version}</docker.tag>  <!-- Tag for this build -->
+        <docker.upstream-registry>${docker.registry}</docker.upstream-registry>  <!-- Registry for base images -->
+        <docker.upstream-tag>${docker.tag}</docker.upstream-tag>  <!-- Tag for base images -->
+        <docker.test-registry>${docker.upstream-registry}</docker.test-registry>  <!-- Registry for integration test dependencies -->
+        <docker.test-tag>${docker.upstream-tag}</docker.test-tag>  <!-- Tag for integration test dependencies -->
+        <dockerfile-maven-plugin.version>1.3.6</dockerfile-maven-plugin.version>
+        <exec-maven-plugin.version>1.5.0</exec-maven-plugin.version>
```

Comment:
if we highlight `docker build support` these following 3 things seem like they might not fit that category? or at least even if they are being pulled in due to it, they would not be specific to it?

Reply:
@aayars @ewencp - I mostly moved this from common-docker over to common.

Reply:
I'd move exec-maven-plugin.version, licenses.version, and slf4j-api.version up to the first group of properties (in alphabetical order) since they're not specific to Docker.